.travis.yml

```diff
@@ -40,7 +40,7 @@ script:
   - pwsh -File tools/travis.ps1
   # spellcheck
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      mdspell '**/*.md' '!powershell/**/*.md' --ignore-numbers --ignore-acronyms --report;
+      mdspell '**/*.md' '!powershell/**/*.md' '!**/publish/**/*.md' --ignore-numbers --ignore-acronyms --report;
```

Comment:
Can `'!powershell/**/*.md'` maybe be removed?

Reply:
Not sure what it was for.

Reply:
I removed it and it appears to not be needed.  I added a comment to why my exclusion is there.